foi-web/forms-flow-ai/forms-flow-web/src/apiManager/endpoints/index.js

```diff
@@ -32,6 +32,8 @@ const API = {
   BPM_FORM_SUBMIT:`${BPM_BASE_URL}/task/<task_id>/submit-form`,
   GET_BPM_FORM_LIST:`${BPM_BASE_URL_EXT}/form`,
   UPDATE_ASSIGNEE_BPM_TASK:`${BPM_BASE_URL}/task/<task_id>/assignee`,
+
+  FOI_GET_REQUESTS_API: 'http://localhost:15000/api/foirawrequests',//`${FOI_API_URL}/requests`,
```

Comment:
Is this meant to be hardcoded to localhost? Any reason it's not using the string interpolation in the comment?

Reply:
@acoard-aot This has been fixed